src/resnet50.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{model::*, input::*};
+use egg::*;
+
+
+fn resnet_block(graph: &mut GraphConverter, mut input: Id, strides: (i32, i32), out_channels: i32, input_dim_1: i32, name_gen: &mut NameGen) -> Id {
+    let w1 = graph.new_input(&name_gen.get_name(), out_channels, input_dim_1, 1, 1);
+    let t = graph.conv2d(input, w1, 1, 1, PSAME, ACTRELU);
```

Comment:
In c++, when it is unclear what the arguments to a function mean, we usually add comments like:
conv2d(input, w1, /*stride_h=*/1, /*stride_w=*/1, /*padding=*/PSAME, /*activation=*/ACTRELU); the argument names should match their definitions. This makes it easy to understand the code without having to refer to the function definition. I'd recommend doing something similar in this resnet_block code. This might make it harder to just copy from TASO, but maybe it is a good way to double check that you build graphs correctly?